nuxt.config.js

```diff
@@ -1,6 +1,7 @@
 
 export default {
   mode: 'universal',
+  components: true,
```

Comment:
Just wondering if this is needed? What is it fixing? Thanks Ruby

Reply:
Cool, sounds good. It will be turned on by default in Nuxt 3. Just wasn't sure if you were fixing a bug or not. Thanks Ruby